Update optimize-slm-guide.mdx (#11759)
newrelic/docs-website: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/optimize-slm-guide.mdx

Divide B / A to calculate revenue leakage % rate.

> ## Tracking your Service Level Objectives

Reviewer: ```suggestion
## Tracking your service level objectives [#tracking-slo]
```

Service levels are a practice, just like testing, alerting, game days etc. You could think of them as a scientific instrument that helps you to measure the "health" of your systems. But like all tools, service levels require calibration.

> Include the service level practice in your team's process. The below recommendations were distilled from the experience using service levels within teams at New Relic, and you should adjust them for your specific team requirements:

Reviewer: ```suggestion
Include the service level practice in your team's process. The following recommendations were distilled from the experience using service levels within teams at New Relic, and you should adjust them for your specific team requirements:
```

- Do a periodic review of the service levels, and pay close attention to:

>     + Do the SLIs reflect incidents & pages?

Reviewer: ```suggestion
    + Do the SLIs reflect incidents and pages?
```

    + What's your error budget for a week?

>         * If it's too low, investigate what caused a drop, use "Analyze" feature to find bad events that caused it

Reviewer: ```suggestion
        * If it's too low, investigate what caused a drop, using the "Analyze" feature to find bad events that caused it
```

>         * If it's too low, investigate what caused a drop, use "Analyze" feature to find bad events that caused it

Reviewer: add some periods to these to match the one with a period

>         * If it's too low, investigate what caused a drop, using the "Analyze" feature to find bad events that caused it

Author: ```suggestion
        * If it's too low, investigate what caused a drop, using the "Analyze" feature to find bad events that caused it,
```

>         * If it's 100%, make sure your indicator is correct and SLO is aggressive enough. Being at 100% indicates the SLO is too safe

Reviewer: ```suggestion
        * If it's 100%, make sure your indicator is correct and the SLO is aggressive enough. Being at 100% indicates the SLO is too safe.
```

>         * What is the trend that you observe in various time periods (1d/7d/28d)

Author: ```suggestion
        * What is the trend that you observe in various time periods (1d/7d/28d).
```

> - Keep an eye on SLIs during game days. SLIs should reflect the impact, just like your alerts do

Author: ```suggestion
- Keep an eye on SLIs during game days. SLIs should reflect the impact, just like your alerts do.
```

> - When you have a drop of error budget on production, evaluate why it didn't happen on staging

Author: ```suggestion
- When you have a drop of error budget on production, evaluate why it didn't happen on staging.
```
